### ISSUE TITLE
Ignore dependency updates for those that are not in the dependency graph

### ISSUE THREAD

**Issue by ngmip** [mojohaus/versions-maven-plugin#437]
Hello,

TL;DR : `versions:dependency-updates-report` reports available updates on libraries that are not in the project's dependency graph.

It seems that the `display-dependency-update` and the `dependency-update-report` are taking all possible dependency managed by the projet or it's parent, even if that dependency is not actually part of the project's dependency graph.

This is a tpyical spring-boot situation where the spring-boot-parent pom manages hundreds of dependencies but only a few are ever added to the projet.
Yet the analyze picks up all these declared dependencies and checks if a new one exists

Any settings/trick to ignore the dependencies that are not part of the dependency graph ?

**Comment by Mattheo1985** [comment]
The experimental parameter [\<processDependencyManagementTransitive\>](https://www.mojohaus.org/versions-maven-plugin/dependency-updates-report-mojo.html#processDependencyManagementTransitive) might help. Unfortunately it's not available in versions:display-dependency-updates (https://github.com/mojohaus/versions-maven-plugin/issues/267 https://github.com/mojohaus/versions-maven-plugin/issues/291).
